shared/util.go

```diff
@@ -1231,7 +1231,7 @@ func RenderTemplate(template string, ctx pongo2.Context) (string, error) {
 	return ret, err
 }
 
-func GetSnapshotExpiry(refDate time.Time, s string) (time.Time, error) {
+func GetExpiry(refDate time.Time, s string) (time.Time, error) {
```

Comment:
Whilst you're looking at this, please could you add a comment on the function explaining what it does, and what `refDate` and what format `s` is? Thanks